### DISCUSSION TITLE
Outbound Requirements for Firewall Traffic

### DISCUSSION THREAD

**Discussion by jordanabakerafs** [Snowflake-Labs/schemachange#382] [Q&A]
** What are you trying to accomplish? **
We currently have an application team implementing some pipelines that are leveraging schemachange to call outbound to our Snowflake org.  Both are located in Azure - GCC High.  I checked the docs but was not able to find a list of outbound network traffic requirements.  Is there a list of outbound network requirements anywhere that could be used for us to validate traffic?

** What options have you tried so far ? **
We can see failed calls outbound to a few URLs on port 80, mostly related to OCSP.  We can infer requirements if needed, but if there is a list somewhere it would greatly accelerate our process.

Thanks!

**Comment by sfc-gh-tmathew** [comment]
This is more of a snowflake specific question than a schemachange question.  Here is the https://docs.snowflake.com/en/sql-reference/functions/system_allowlist snowflake documentation for reference.

If you resolved this and do not need assistance, please close this thread at your convenience. Apologies for not getting back sooner.
